packages/client/lib/blockchain/chain.ts

```diff
@@ -103,14 +103,19 @@ export class Chain extends EventEmitter {
     super()
 
     this.config = options.config
+    let validateConsensus = false
+    if (this.config.chainCommon.consensusAlgorithm() === 'clique') {
+      validateConsensus = true
+    }
 
     this.blockchain =
       options.blockchain ??
       new Blockchain({
         db: options.chainDB,
         common: this.config.chainCommon,
+        hardforkByHeadBlockNumber: true,
         validateBlocks: true,
-        validateConsensus: true,
+        validateConsensus,
```

Comment:
I know we had this discussion earlier, but why disable the consensus validation? Now everyone can feed us bogus PoW blocks which we will then execute. I understand that the PoW validation takes time, but we should really validate it in our client.

Reply:
could we solve this by using [node-ethash](https://github.com/ethereumjs/node-ethash#performance-node-ethash-vs-ethashjs) in the client?

Reply:
Hmm that's interesting, I didn't even know we had this. The performance boost is insane, we should definitely look into this (possibly we should create a general Ethash interface if this is not yet done, and then let our users choose downstream if they want to use Ethashjs or node-ethash)

Reply:
I have no idea in what state the `note-ethash` library is actually. Will open a dedicated issue on this.